.github/workflows/docker.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: prismarinejs/flying-squid
+          path: deploy
+          registry: docker.io
+          dockerfile: deploy/Dockerfile
+          tag_with_ref: true
+
+      - name: Push to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          repository: prismarinejs/flying-squid/flying-squid
```

Comment:
Not  sure about this one though, don't know Github Package Docker registry, they say to name "owner/repo/image" unlike DockerHub "owner/image"